webapp/src/containers/modals/SubscribeModal/filters/pipelines.tsx

```diff
@@ -0,0 +1,215 @@
+import React, {useCallback, useEffect, useMemo} from 'react';
+
+import pluginConstants from 'pluginConstants';
+import {filterLabelValuePairAll} from 'pluginConstants/common';
+
+import Dropdown from 'components/dropdown';
+
+import usePluginApi from 'hooks/usePluginApi';
+
+import {formLabelValuePairs} from 'utils';
+
+type PipelinesFilterProps = {
+    organization: string
+    projectId: string
+    eventType: string
+    selectedBuildPipeline: string
+    handleSelectBuildPipeline: (value: string, name?: string) => void
+    setIsFiltersError: (value: boolean) => void
+    selectedBuildStatus: string
+    handleSelectBuildStatus: (value: string, name?: string) => void
+    handleSelectReleasePipeline: (value: string, name?: string) => void
+    selectedReleasePipeline: string
```

Comment:
```suggestion
    handleSelectReleasePipeline: (value: string, name?: string) => void
    selectReleasePipeline: string
```

Reply:
shouldn't this be like this or vice-versa? if yes, check everywhere

Reply:
No, we are storing the selected value in this variable and `handleSelectReleasePipeline` is used when we select a value from the dropdowns